CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -45,6 +45,7 @@ function PlayerHospital:PlayerHospital(world, avail_rooms, name)
   }
 
   self.win_declined = false -- Has not yet declined the level win fax
+  self.announce_vip = 0 -- Whether there is a VIP who needs to be announced
```

Comment:
Predates you, but using 0 and -1 seems sloppy in practice.

Reply:
Because this wasn't a boolean, it's a counter, every time a VIP is added that should be announced it's incremented and when they are announced it's decremented; presumably in case multiple VIPs arrive on the same day?

Reply:
Yes it's a good point. Not that more than one should be arriving together